docs/Documentation/UserGuideV0.7.0/0-Content.md

```diff
@@ -0,0 +1,29 @@
+# Chapter 1: Overview
```

Comment:
Hi, this file also needs Apache Header. 
You'd better run `mvn apache-rat:check` before you submit PR.